pkg/bmc/ipmi_noop.go

```diff
@@ -0,0 +1,98 @@
+package bmc
+
+import (
+	"net/url"
+)
+
+func init() {
+	RegisterFactory("ipmi-noop", newIPMINoopAccessDetails, []string{})
+	RegisterFactory("libvirt-noop", newIPMINoopAccessDetails, []string{})
+}
+
+func newIPMINoopAccessDetails(parsedURL *url.URL, disableCertificateVerification bool) (AccessDetails, error) {
+	return &ipmiNoopAccessDetails{
+		bmcType:                        parsedURL.Scheme,
+		portNum:                        parsedURL.Port(),
+		hostname:                       parsedURL.Hostname(),
+		disableCertificateVerification: disableCertificateVerification,
+	}, nil
+}
+
+type ipmiNoopAccessDetails struct {
+	bmcType                        string
+	portNum                        string
+	hostname                       string
+	disableCertificateVerification bool
+}
+
+const ipmiDefaultPort = "623"
```

Comment:
Can we move it to a common location? maybe somewhere under pkg/provisioner/ironic?